### PR TITLE
doc: Central HIDs - improve sample documentation

### PR DESCRIPTION
**Pull request by grochu**
Clarified documentation of printed output when enabling Caps lock.

### CODE REVIEW

**Comment by grochu** [comment]
@b-gent could you review?

File: samples/bluetooth/central_hids/README.rst [hunk 1]
```diff
@@ -93,10 +93,13 @@ Testing with another board
 
 
 #. Press Button 1 and Button 3 on the Central HIDS board and observe that LED 1 on the keyboard board changes its state.
```
Comment (b-gent) [inline samples/bluetooth/central_hids/README.rst]: In all these steps we say "press Button x and Button y on the board (...)".
It seems slightly confusing to me, my first understanding is to press them both at the same time while they should clearly be pressed consecutively. I will try to find better wording for this.

Comment (grochu) [reply samples/bluetooth/central_hids/README.rst]: Yes, it may be worth re-phrasing.

Comment (grochu) [reply samples/bluetooth/central_hids/README.rst]: In this case these Buttons can be pressed consecutively in any order as they have the same effect from the user point of view - they just use different GATT procedures to update the characteristic.

**Comment by grochu** [comment]
I squashed the commits. Ready to merge?

**Comment by b-gent** [comment]
@grochu  yes, looking good

**Pull request merged**